### PR TITLE
ci(gitlab): use external gitlab-ci-yaml gitlab.com project

### PR DESCRIPTION
**Pull request by douglasduteil**

### CODE REVIEW

File: .gitlab-ci/stages/07-notify.yml [hunk 1]
```diff
@@ -20,52 +25,47 @@ Notify Success:
...
-#      https://gitlab.factory.social.gouv.fr/api/v4/projects/51/trigger/pipeline
+#
+
+Production:
+  stage: "Deploy"
+  image: curlimages/curl:7.67.0
+  dependencies: []
+  when: manual
+  only:
+    - branches
```
Comment (lionelB) [inline .gitlab-ci/stages/07-notify.yml]: only tags ?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
   - /.gitlab-ci/stages/04-scan.yml
-  - /.gitlab-ci/stages/05-register-data.yml
-
   - /.gitlab-ci/stages/06-deploy.yml
-
   - /.gitlab-ci/stages/07-notify.yml
 
-
 stages:
   - "Prepare"
```
Comment (lionelB) [inline .gitlab-ci.yml:21]: Prepare come from where ?

**Review by lionelB — APPROVED** [review]
looks good

File: .gitlab-ci/stages/07-notify.yml (hunk 1 above)
Comment (douglasduteil) [reply .gitlab-ci/stages/07-notify.yml]:
```suggestion
    - tags
```

File: .gitlab-ci.yml (hunk 2 above)
Comment (douglasduteil) [reply .gitlab-ci.yml:21]: The `stages` entry defines what stages in what order are possible.
Preprare like before in a possible stage

**Review by lionelB — APPROVED** [review]

**Pull request merged**